docs/sources/shared/datasources/sql-query-builder-macros.md

```diff
@@ -0,0 +1,19 @@
+---
+headless: true
+labels:
+  products:
+    - enterprise
+    - oss
+---
+
+#### Macros
```

Comment:
Note if you need to include this in places where it may not be an H4, you can use the `leveloffset` parameter to offset the heading: https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsshared. Only positive offsets are implemented at the moment so it may make sense to start this as an H1. Or you can ask me to do the work to implement the negative offsets :D

Reply:
Good to know, thanks Jack.